capty.py

```
@@ -109,79 +346,223 @@ def clear_rect(self):
+        self.settings_popup.add(settings_vbox)
+        
+        # Settings title
+        title_label = Gtk.Label(label="<b>Recording Settings</b>", use_markup=True)
+        title_label.get_style_context().add_class("settings-title")
+        settings_vbox.pack_start(title_label, False, False, 0)
+        
+        # Filename
+        filename_box = Gtk.Box(orientation=Gtk.Orientation.HORIZONTAL, spacing=8)
+        filename_box.pack_start(Gtk.Label(label="Filename:"), False, False, 0)
+        self.filename_entry = Gtk.Entry()
+        self.filename_entry.set_text(f"capty")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)